Sources/XCTest/XCAbstractTest.swift

```diff
@@ -26,6 +26,37 @@ public class XCTest {
         fatalError("Must be overridden by subclasses.")
     }
 
+    /// The `XCTestRun` subclass that will be instantiated when the test is run
+    /// to hold the test's results. Must be overridden by subclasses.
+    public var testRunClass: AnyClass {
+        fatalError("Must be overridden by subclasses.")
+    }
+
+    /// The test run object that executed the test, an instance of
+    /// testRunClass. If the test has not yet been run, this will be nil.
+    /// - Note: FIXME: This property is meant to be `internal(set)`. It is
```

Comment:
Curiosity: why would this (ideally) be `internal(set)` as opposed to `private(set)`?